test/snapshot-order/randomness.js

```diff
@@ -0,0 +1,31 @@
+const test = require('@ava/test');
+const exec = require('../helpers/exec');
+const fs = require('fs');
+const path = require('path');
+const getSnapshotIds = require('./helpers/get-snapshot-ids');
+
+test('deterministic and sorted over a large, random test case', async t => {
+	const options = {
+		cwd: exec.cwd('randomness'),
+		env: {
+			AVA_FORCE_CI: 'not-ci'
+		}
+	};
+
+	const snapshotPath = path.join(options.cwd, 'test.js.snap');
+	const reportPath = path.join(options.cwd, 'test.js.md');
+
+	// Run test
+	await exec.fixture(['--update-snapshots'], options);
+
+	// Assert snapshot is unchanged
+	const snapshot = fs.readFileSync(snapshotPath);
+
+	t.snapshot(snapshot);
```

Comment:
Let's add a message so it's a little clearer what is in this snapshot. Because if we change anything else in the snapshot format, this snapshot needs updating too.

```suggestion
	t.snapshot(snapshot, 'resulting snapshot in binary encoding');
```